src/pfs/pfs.proto

```diff
@@ -131,36 +131,25 @@ message CommitInfo {
   google.protobuf.Timestamp started = 6;
   google.protobuf.Timestamp finished = 7;
   uint64 size_bytes = 8;
-
-  // the commits and their original branches on which this commit is provenant
-  repeated CommitProvenance provenance = 9;
-
-  // ReadyProvenance is the number of provenant commits which have been
-  // finished, if ReadyProvenance == len(Provenance) then the commit is ready
-  // to be processed by pps.
-  int64 ready_provenance = 10;
-
-  repeated CommitRange subvenance = 11;
-
-  int64 subvenant_commits_success = 12;
-  int64 subvenant_commits_failure = 13;
-  int64 subvenant_commits_total = 14;
 }
 
 message Job {
   string id = 1 [(gogoproto.customname) = "ID"];
 }
 
+message JobCommitInfo {
+  Branch branch = 1;
+  repeated Branch provenance = 2;
+}
+
```

Comment:
So the name for this seems kind of weird, my understanding is that this structure is being used to record the link between a branch and its provenance for a specific job. With that in mind I feel like there's a few options for renaming this:

1. We could just use the `BranchInfo` msg instead, it has a few extra fields we don't want so we could just blank those. Not sure how ugly you feel that is.
2. We could just use an embedded msg in the msgs below, it seems like that's the only place it's used.
3. We could call it something like `BranchProvenance` since it seems like its main job is recording that.

Reply:
`BranchProvenance` sounds good.  Like we discussed, this will be changed to specify `DirectProvenance` for each commit instead of the full provenance.  Also, this structure shouldn't really be returned directly to users, we'll return a hydrated structure with `CommitInfos` in it as well (and possibly an expanded `Provenance` graph).

Reply:
I ended up using `BranchInfo` and clarifying that we only set the `Branch` and `DirectProvenance` fields - it seems like proto3 is pretty good about making this efficient